components/TextFields/src/ContainedInputView/MDCBaseTextField.m

```diff
@@ -205,6 +231,16 @@ - (MDCBaseTextFieldLayout *)calculateLayoutWithTextFieldSize:(CGSize)textFieldSi
                                     numberOfTextRows:1];
 }
 
+- (CGFloat)normalizedCustomAssistiveLabelDrawPriority:(CGFloat)customPriority {
```

Comment:
s/nomralized/clamp/

This is more often called clamping
https://en.m.wikipedia.org/wiki/Clamping_(graphics)

Reply:
Ahhhh nice, thank you!